### PR TITLE
BIP 8: Version bits with optional guaranteed lock-in

### PR DESCRIPTION
**Pull request by shaolinfry**

### CODE REVIEW

**Comment by shaolinfry** [comment]
Updated state diagram.

**Comment by shaolinfry** [comment]
What is the BIP number for this proposal?

**Comment by luke-jr** [comment]
Needs to address Backward Compatibility.

And the flowchart thing is still buggy - it doesn't check the flag before failing it. Personally, I'd just get rid of the flag and say "if it's BIP 8, it never fails".

File: bip-0008.mediawiki [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<pre>
+  BIP: 8
+  Title: Version bits extension with guaranteed lock-in
+  Author: Shaolin Fry <shaolinfry@protonmail.ch>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-????
```
Comment (luke-jr) [inline bip-0008.mediawiki]: ^

**Comment by luke-jr** [comment]
README needs the updated title

File: bip-0008.mediawiki [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+==Deployments==
+
+A living list of deployment proposals can be found [[bip-0008/assignments.mediawiki|here]].
+
+==References==
+
+[[bip-0009.mediawiki|BIP9]]
+
+[https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-February/013643.html Mailing list discussion]
```
Comment (Dsmccarty78) [inline bip-0008.mediawiki]: Who do I get in touch with, if I believe the guy I met years ago and handed him my phone ACTUALLY did what he said he was going to do??? Am I the first transactions after the Genesis block?

**Pull request merged**